cx-sig-charter-draft.md

```diff
@@ -0,0 +1,110 @@
+# Contributor Experience Special Interest Group Charter
+
+This charter adheres to the conventions described in the [Spinnaker Charter README] and uses the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The [Contributor Experience Special Interest Group] (SIG) is responsible for improving the experience of those who upstream contribute to the Spinnaker project. We do this by creating, and maintaining programs and processes that promote community health and reduce project friction while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed.
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or [GitHub issues], we take in the feedback and turn it into our [project list]. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Establish policies, standards, and procedures for the use, [moderation], and management of all public platforms officially used by the project, including but not limited to:
+  - [discuss.spinnaker.io]
+  - [GitHub Management]
+  - [Mailing lists] / Google groups for the project as a whole (eg: <need community mailing list>@googlegroups.com) and for individual sigs and working groups where the Chairs have provided us ownership
+  - [Slack]
+  - [/spinnaker] YouTube channel
+  - [Zoom]
+- Establish and staff teams responsible for the administration and moderation of these platforms
+  - Teams must be staffed by trusted contributors spanning time zones, see [moderation] for more detail
```

Comment:
`trusted` may need to be changed. If not, we need to define what `trusted` is defined as.